src/repl/scala/tools/nsc/interpreter/ILoop.scala

```diff
@@ -214,16 +186,16 @@ class ILoop(in0: Option[BufferedReader], protected val out: JPrintWriter)
     cmd("implicits", "[-v]", "show the implicits in scope", intp.implicitsCommand),
     cmd("javap", "<path|class>", "disassemble a file or class name", javapCommand),
     cmd("line", "<id>|<line>", "place line(s) at the end of history", lineCommand),
-    cmd("load", "<path>", "interpret lines in a file", loadCommand),
-    cmd("paste", "[-raw] [path]", "enter paste mode or paste a file", pasteCommand),
+    cmd("load", "<path>", "interpret lines in a file", loadCommand, fileCompletion),
```

Comment:
A `LoopCommand` is only `String => Result` so all the config is externalized here. The list of commands here could be rewritten as a list of `SomeCommand`, or pairs of `String -> SomeCommand`. That is worth doing when REPL is rewritten.